cliquet/__init__.py

```diff
@@ -188,6 +191,13 @@ def includeme(config):
     handle_api_redirection(config)
     config.add_tween("cliquet.end_of_life_tween_factory")
 
+    if settings.get('cliquet.sentry_dsn') is not None:
```

Comment:
What happens if I have ``cliquet.sentry_dsn = server.com`` in my .ini and if run with ``CLIQUET_SENTRY_DSN= make serve`` ?